news/2022-06-30-community-contributors-2021.md

```diff
@@ -0,0 +1,101 @@
+
+Some people take it upon themselves to go simply above and beyond all expectations, and it is upon their labours that the foundation of osu! has been laid all this time.
+
+![Community Contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg)
+
+We are privileged to recognise more of these incredible people (in no particular order):
+
+## Contributors
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/kasumi-sama.jpg)](https://osu.ppy.sh/users/6177263)
+
+Single-handedly leading the charge forward to elevate the osu!taiko community tournament scene, Kasumi-sama has created a towering legacy under the Global Taiko Showdown (GTS) series. Renowned for their excellent production quality, competitive spirit and considerable reach into entry-level tournaments as a whole, GTS has rapidly established itself as one of the most robust and engaged community projects seen in recent years.
+
+Paired with a keen artistic drive and the wherewithal to engage with many of our Featured Artists and mappers alike to produce new and original content for the community, it is safe to say that Kasumi-sama represents the latest generation of a new era of osu! creators wading knee deep into osu!'s most prominent ethic: making things more awesome.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/frenzibyte.jpg)](https://osu.ppy.sh/users/14210502)
+
+A regular contributor to ongoing development efforts in osu!(lazer), frenzibyte has made himself well-known in the scene within recent years. His body of work is vast and difficult to accurately explain in a mere few paragraphs, best described instead as making efforts like the iOS version of the game possible once again for thousands of people to enjoy.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/susko3.jpg)](https://osu.ppy.sh/users/18945305)
+
+Tirelessly striving to fix bugs and bridge the differences between computing platforms, Susko3’s efforts in lazer development have proven instrumental in pushing the project forward in recent times. Their expertise in both lazer’s underlying development framework and other libraries (such as SDL) has proven integral to ongoing projects, and we are tremendously thankful for their dedication and support.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/stanr.jpg)](https://osu.ppy.sh/users/7217455)
+
+A mainstay presence in the performance points development community, StanR has been at the forefront of making meaningful changes to osu!'s key performance algorithms for a considerable length of time at this point, contributing new changes of his own and vetting those made by others. While this may not sound like much at first glance, his contributions represent hundreds of hours of work and have manifested in scoring becoming fairer and more fun for everyone.
```

Comment:
just want to double-check why "While this may not sound like much at first glance" was necessary to say, because to me it kind of does sound like a large effort. is this representing an opinion ppl would actually have?

Reply:
good call, will remove